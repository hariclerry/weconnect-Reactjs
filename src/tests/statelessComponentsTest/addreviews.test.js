import React from "react";
import {shallow, mount} from "enzyme";
import ReviewBusiness from "../../components/business/reviews"





describe('ReviewBusiness', ()=>{
  let wrapper;
  beforeEach(()=>{
    const reviews = []
    const addReview = {}
    const businessName = {}
    wrapper = shallow(<ReviewBusiness
                        reviews={reviews} addReview = {addReview}
                        businessName={businessName} />)
  })

  it('should match snapshot', () =>{
    expect(wrapper).toMatchSnapshot();
  })
})



  



