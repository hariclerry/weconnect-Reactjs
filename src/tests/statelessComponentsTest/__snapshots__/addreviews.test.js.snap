// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewBusiness should match snapshot 1`] = `
<div>
  <withRouter(Connect(NavBar)) />
  <div
    className="container"
  >
    <div
      style={
        Object {
          "margin": "20px",
        }
      }
    >
      <div
        className="box"
      >
        <h1 />
        <p>
          <b>
            Category: 
          </b>
           
        </p>
        <p>
          <b>
            Location: 
          </b>
        </p>
        <p>
          <b>
            Description: 
          </b>
        </p>
      </div>
    </div>
    <hr />
  </div>
  <div
    style={
      Object {
        "margin": "50px 150px",
      }
    }
  >
    <form
      onSubmit={Object {}}
    >
      <div
        className="form-group"
      >
        <label
          className="control-label "
          htmlFor="description"
        >
          Say Something about this Business:
        </label>
        <textarea
          className="form-control"
          cols="2"
          id="description"
          name="description"
          required={true}
          rows="2"
        />
      </div>
      <div
        className="form-group"
      >
        <button
          className="btn btn-primary "
          name="submit"
          type="submit"
        >
          Add Review
        </button>
      </div>
    </form>
  </div>
  <div
    style={
      Object {
        "margin": "50px 150px",
        "width": "auto",
      }
    }
  >
    <h3>
      Reviews
    </h3>
  </div>
  <Footer />
</div>
`;
